src/components/icons/pages/icons.jsx

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import DocsBlock from 'components/DocsBlock';
 import ContrastBox from 'components/ContrastBox';
-import Icon, {TYPE, ICON_COLOR, SIZE} from '../Icon';
+import Icon, {TYPE, STD_TYPE, ICON_COLOR, SIZE} from '../Icon';
+import Text from '../../text/Text';
 
 const icons = () => (
   <div>
     <DocsBlock>
       <ContrastBox>
+        <Text size="small">This set of the icons is deprecated</Text>
```

Comment:
hm, maybe we could make it a bit more visible, wdyt?

Reply:
maybe just a white color since background is dark?

Reply:
Done, I move it above the `ContrastBox` and change the color of the text, now it's peach and on the white background.